frontends/web/src/routes/device/bitbox02/settings.tsx

```diff
@@ -53,8 +48,10 @@ class Settings extends Component<Props, State> {
     }
 
     private getInfo = () => {
-        apiGet(this.apiPrefix() + '/info').then(response => {
-            this.setState({ deviceInfo: response.deviceInfo });
+        getDeviceInfo(this.props.deviceID).then(response => {
+            if (response.success) {
```

Comment:
maybe not now, but can think about making these api calls reject() instead of resolve(), so one can skip the success check here and use `.catch()` to handle errors. it's a bigger story  to think about though.

Reply:
agree that would be nice, lets discuss with @x1ddos but not part of this PR.

`.catch()` would also catch if there was a real error, without success:false.

Reply:
Discussed with @x1ddos and moved the success check so it is api internal and just return the deviceInfo  https://github.com/digitalbitbox/bitbox-wallet-app/pull/1445/commits/9eb6607f64abb0106ce8e154ac59faab236efab4